allPostsToWordpress.py

```diff
@@ -21,7 +21,12 @@ def addPost(post, channel, username, wpUrl, doProtected, protectedPassword, doCo
     ET.SubElement(item, 'wp:post_date_gmt').text = pubDate.astimezone(datetime.timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
     ET.SubElement(item, 'wp:comment_status').text = 'open'
     ET.SubElement(item, 'wp:ping_status').text = 'open'
-    ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
+    postTitle = post.find('title').text.strip().lower()
+    wordChar = re.sub('[^\s\w]', '', postTitle)
+    if postTitle == '(no subject)':
+        ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
```

Comment:
Oh, sorry - no, I didn't write a code snippet for that.  What you'll want to do is use the if/else clause to assign the name of the post we'd like into a variable (something like `postName`), then use that.

This is better style because it means you don't have to duplicate the `ET.SubElement(item, 'wp:post_name').text` assignment.